src/AccessMessageMetadata/ADAr/ADAr_Extractor.h

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This code has been produced by the European Spallation Source
+// and its partner institutes under the BSD 2 Clause License.
+//
+// See LICENSE.md at the top level for license information.
+//
+// Screaming Udder!                              https://esss.se
+
+/** Copyright (C) 2018 European Spallation Source ERIC */
+
+/// \file
+/// \brief Define classes required to implement the ADC file writing module.
+
+#pragma once
+
+#include "FlatbufferReader.h"
+
+namespace AccessMessageMetadata {
+using FlatbufferMessage = FileWriter::FlatbufferMessage;
+using FBReaderBase = FileWriter::FlatbufferReader;
```

Comment:
`using FBReaderBase` seems overkill for one use of the class, but obviously not a big deal.

Reply:
Fair comment, I think it is the result of this PR largely being copy paste code.